presto-native-execution/presto_cpp/main/http/HttpServer.cpp

```diff
@@ -75,6 +78,16 @@ void sendErrorResponse(
       .sendWithEOM();
 }
 
+void sendErrorJsonResponse(
```

Comment:
Both sendErrorJsonResponse and sendOkResponse have very similar ResponseBuilder usage except for the status. Can we abstract a single common method for sendResponse and have a parameter for status.. then sendOkResponse and sendErrorResponse can use it with their specific status ?

Reply:
I don't like adding a separate sendErrorJsonResponse, so I'll give it another shot to clean it up with a general sendResponse

Reply:
updated, it's a little bit cleaner with a general sendResponse